assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -24,4 +24,4 @@ tags:
   license: apache-2.0
   task: text-generation
   author: mistralai
-version: 5
+version: 6
```

Comment:
version 5 is not published yet. Or was there a partial release done last time for this model? 
Could you check by running: 

az ml model show for version 6 once?